README.md

```diff
@@ -23,10 +23,7 @@ The app requires:
 To start the main services excluding the use of force app: 
 `docker-compose up --scale=use-of-force=0`
 
-To start all services including use of force and new-nomis: 
-`docker-compose -f docker-compose.yml -f docker-compose.newnomis.yml up`
```

Comment:
this was quite out of date and not super useful so rather than fix I just got rid.